### PR TITLE
uboot-kirkwood: update to 2020.04

### PR DESCRIPTION
**Pull request by CHKDSK88**
Update U-Boot to current 2020.04 release for kirkwood platform.

Catch up with upstream and move some configuration options from
the header files to the corresponding defconfig files.

Compile tested: all devices
Run tested: nsa310, pogoplugv4

Tested-by: Cezary Jackiewicz <cezary@eko.one.pl> [nsa310]
Signed-off-by: Pawel Dembicki <paweldembicki@gmail.com>

### CODE REVIEW

**Comment by CHKDSK88** [comment]
@bobafetthotmail 
Could You review and test?

**Comment by CHKDSK88** [comment]
Runtested: pogoplug-v4

File: package/boot/uboot-kirkwood/Makefile [hunk 1]
```diff
@@ -1,16 +1,16 @@
 #
-# Copyright (C) 2010-2014 OpenWrt.org
+# Copyright (C) 2010-2020 OpenWrt.org
```
Comment (diizzyy) [inline package/boot/uboot-kirkwood/Makefile:2]: Don't do that
https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain

**Comment by CHKDSK88** [comment]
Tested-by: Cezary Jackiewicz <cezary@eko.one.pl>

**Comment by ynezz** [comment]
> Tested-by: Cezary Jackiewicz [cezary@eko.one.pl](mailto:cezary@eko.one.pl)

Did @obsy lost his access to GitHub, that he cant provide this tag himself? :)

**Comment by obsy** [comment]
No, just Pawel was faster :). Tested on ZyXEL NSA-310

Tested-by: Cezary Jackiewicz <cezary@eko.one.pl>

**Comment by ynezz** [comment]
Thanks! Pulled into my staging tree at https://git.openwrt.org/openwrt/staging/ynezz.git

**Comment by CHKDSK88** [comment]
@ynezz

Please consider backport it to stable. At this moment kirkwood u-boot in 19.07 have broken usb and Ethernet. 

@obsy could confirm it.

**Comment by obsy** [comment]
```
scanning bus 0 for devices... EHCI timed out on TD - token=0x80008d80
EHCI timed out on TD - token=0x80008c80
EHCI timed out on TD - token=0x80008c80
EHCI timed out on TD - token=0x80008c80
 ERROR: NOT USB_CONFIG_DESC 80
EHCI timed out on TD - token=0x80008d80
EHCI timed out on TD - token=0x80008c80
EHCI timed out on TD - token=0x80008c80
```

**Comment by ynezz** [comment]
> Please consider backport it to stable.

Nope, this is not going to happen.

> At this moment kirkwood u-boot in 19.07 have broken usb and Ethernet.

Then I'm afraid, that it has to be fixed with some patches on top of the U-Boot in 19.07.

**Pull request closed without merging**